sdk/healthinsights/azure-health-insights-radiologyinsights/src/test/java/com/azure/health/insights/radiologyinsights/RadiologyInsightsFindingTest.java

```diff
@@ -0,0 +1,109 @@
+                + "\r\n "
+                + "\r\nCOMPARISON:   "
+                + "\r\nRight upper quadrant sonographic performed 1 day prior."
+                + "\r\n "
+                + "\r\nTECHNIQUE:   "
+                + "\r\nTransabdominal grayscale pelvic sonography with duplex color Doppler "
+                + "\r\nand spectral waveform analysis of the ovaries."
+                + "\r\n "
+                + "\r\nFINDINGS:   "
+                + "\r\nThe uterus is unremarkable given the transabdominal technique with "
+                + "\r\nendometrial echo complex within physiologic normal limits. The "
+                + "\r\novaries are symmetric in size, measuring 2.5 x 1.2 x 3.0 cm and the "
+                + "\r\nleft measuring 2.8 x 1.5 x 1.9 cm.\n \r\nOn duplex imaging, Doppler signal is symmetric."
+                + "\r\n "
+                + "\r\nIMPRESSION:   "
+                + "\r\n1. Normal pelvic sonography. Findings of testicular torsion."
+                + "\r\n\nA new US pelvis within the next 6 months is recommended."
+                + "\n\nThese results have been discussed with Dr. Jones at 3 PM on November 5 2020.\n "
+                + "\r\n";
+        setDocumentContent(documentContent);
+        setInferenceType(RadiologyInsightsInferenceType.FINDING);
+        setOrderCode("USPELVIS");
+        setOrderDescription("US PELVIS COMPLETE");
+        
+        try {
+            testRadiologyInsightsWithResponse(request -> {
+                RadiologyInsightsJob riResponse = setPlaybackSyncPollerPollInterval(
+                        getClient().beginInferRadiologyInsights(/*"job" + new Date().getTime()*/"job1715007545785", request)).getFinalResult();
+
+                List<RadiologyInsightsPatientResult> patients = riResponse.getResult().getPatientResults();
+                assertEquals(1, patients.size());
+                
+                RadiologyInsightsPatientResult patient = patients.get(0);
+                List<RadiologyInsightsInference> inferences = patient.getInferences();
+                assertEquals(7, inferences.size());
+                
+                RadiologyInsightsInference inference = inferences.get(0);
+                assertTrue(inference instanceof FindingInference, "Inference should be an instance of FindingInference");
+                
+                FindingInference findingInference = (FindingInference) inference;
+
+                FhirR4Observation finding = findingInference.getFinding();
+//                System.out.println("   Code: ");
+                FhirR4CodeableConcept code = finding.getCode();
+//                displayCodes(code, 2);
+//                System.out.println("   Interpretation: ");
+                List<FhirR4CodeableConcept> interpretationList = finding.getInterpretation();
+//                if (interpretationList != null) {
+//                    for (FhirR4CodeableConcept interpretation : interpretationList) {
```

Comment:
Delete these commented lines, and add assertions instead of printing out the result?

Reply:
I removed those lines as they are not needed any more